env.example

```diff
@@ -0,0 +1,11 @@
+# NODE_ENV should be one of "development" or "production"
+NODE_ENV=development
+
+# PORT is the port used by the web server
+PORT=8080
+
+# NODEMAILER_USERNAME is sender's username credential
+NODEMAILER_USERNAME=
+
+# NODEMAILER_PASSWORD is sender's password credential
+NODEMAILER_PASSWORD=
```

Comment:
Let's remove the NODE_EMAILER_* config until we need it (e.g., future PR can add it if necessary)

Reply:
removed NODE_EMAILER_* config